tests/post/features/backup.feature

```diff
@@ -0,0 +1,5 @@
+@post @ci @local @backup
+Feature: Backup
+    Scenario: Backup multiple times
+        When we run the backup script 6 times
+        Then we have 5 backups on each node
```

Comment:
I think there is an issue somewhere in the test (not sure where) because, as is, it cannot work in multi node to me, since the Job to replicate and clean are asynchronous and you do not wait for them to complete here

Reply:
I think it may be enough to add a simple wait here maybe 1 minute or two

Reply:
Why ? It's pretty simple to wait for Job to succeed (and it's a better approach that arbitrary waiting time)